deployment/helm/synopsys-alert/values.yaml

```diff
@@ -41,6 +41,29 @@ cfssl:
   securityContext: {}
   podSecurityContext: {}
 
+# postgres - configurations for the postgres Pod
+postgres:
+  registry: "docker.io/centos" # override the docker registry at container level
+  isExternal: true # false for running Postgres as a container and true for using External Postgres database
+  host: "localhost" # required only for external postgres, for postgres as a container, it will point to <name>-postgres
```

Comment:
Whether it needs to be empty?

Reply:
If you make it empty then you can use this to require users to set it if isExternal is true
```
{{ required "must provide --set postgres.host=\"\"" .Values.postgres.host }}
```

Reply:
I will add this

Reply:
With the required there the postgres.isExternal must be false by default.  Which is to create a postgres container in the deployment.  That will be the default for alert.